### PR TITLE
Cleanup LICENSE.md completed

### PR DESCRIPTION
**Pull request by eputnam**
Completed [cleanup of LICENSE.md](https://github.com/AnalyticalGraphicsInc/cesium/issues/3234). This is ready.

### CODE REVIEW

**Comment by pjcozzi** [comment]
Thanks @eputnam!

@mramato do you want to review this?  Or should I?

File: LICENSE.md [hunk 1]
```diff
@@ -284,7 +283,7 @@ The Cesium example applications include the following third-party libraries and
 
 http://dojotoolkit.org/
 
-> Copyright (c) 2005-2011, The Dojo Foundation
+> Copyright (c) 2005-2013, The Dojo Foundation
```
Comment (mramato) [inline LICENSE.md]: We just upgraded, so this is actually 2015 now.

File: LICENSE.md [hunk 2]
```diff
@@ -47,11 +47,10 @@ http://hg.grauw.nl/grauw-lib
...
 https://github.com/cujojs/when
 
-> Open Source Initiative OSI - The MIT License
+>  Licensed under the MIT License at:
+>  http://www.opensource.org/licenses/mit-license.php
 >
-> http://www.opensource.org/licenses/mit-license.php
->
-> Copyright (c) 2011 Brian Cavalier
+>  MIT License (c) copyright B Cavalier &amp; J Hann
```
Comment (mramato) [inline LICENSE.md]: Do you have a link to this info?  Going by https://github.com/cujojs/when/blob/master/LICENSE.txt I'm not sure anything needed to change here.

**Comment by mramato** [comment]
Just those 2 comments.  Thanks @eputnam!

**Comment by pjcozzi** [comment]
@eputnam let us know if you are able to make these changes today, and then we would be able to release this with Cesium 1.16 tomorrow.

**Comment by eputnam** [comment]
@pjcozzi , so sorry, i will get these in today.

File: LICENSE.md (hunk 2 above)
Comment (eputnam) [inline LICENSE.md]: oh oops, I was going by what was in Cesium: 
https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/ThirdParty/when.js

**Comment by pjcozzi** [comment]
Awesome, thanks @eputnam!

**Comment by eputnam** [comment]
I'll go back over everything and make sure I've got the info right based on the packages' copyright info, not what's in the Cesium stuff.

**Comment by pjcozzi** [comment]
@eputnam is this ready now?  I see you made a commit to this branch.

File: LICENSE.md (hunk 2 above)
Comment (mramato) [inline LICENSE.md]: Ahh, that makes sense.  Nice catch.

Comment (eputnam) [inline LICENSE.md]: yep! sorry for any confusion, guys. this is my first contribution ever. in
the universe.

**Comment by eputnam** [comment]
yep! sorry for any confusion, guys. this is my first contribution ever. in
the universe.

**Comment by mramato** [comment]
No worries @eputnam.  Usually when any review comments have been address, it's good to add a "ready" message so that we know to look at it again.  Otherwise we might merge it before you're done with whatever changes you wanted to make.  In this case, this looks great and congratulations on your first contribution in the universe.  Hopefully one of many!

**Comment by pjcozzi** [comment]
> yep! sorry for any confusion, guys. this is my first contribution ever. in the universe.

Wow, very cool.  Thanks for the contribution!  What do you want to do next? :) https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#getting-started-contributing-code-and-documentation

By the way, are you also @eputnam on twitter?

**Pull request merged**